docs/release/fundamentals/tables-and-data-operations.ipynb

```diff
@@ -0,0 +1,2565 @@
+       "      <td id=\"T_6f49e_row2_col1\" class=\"data row2 col1\" >string</td>\n",
+       "      <td id=\"T_6f49e_row2_col2\" class=\"data row2 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f49e_row3_col0\" class=\"data row3 col0\" >timestamp</td>\n",
+       "      <td id=\"T_6f49e_row3_col1\" class=\"data row3 col1\" >timestamp</td>\n",
+       "      <td id=\"T_6f49e_row3_col2\" class=\"data row3 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f49e_row4_col0\" class=\"data row4 col0\" >longitude</td>\n",
+       "      <td id=\"T_6f49e_row4_col1\" class=\"data row4 col1\" >float</td>\n",
+       "      <td id=\"T_6f49e_row4_col2\" class=\"data row4 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f49e_row5_col0\" class=\"data row5 col0\" >latitude</td>\n",
+       "      <td id=\"T_6f49e_row5_col1\" class=\"data row5 col1\" >float</td>\n",
+       "      <td id=\"T_6f49e_row5_col2\" class=\"data row5 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f49e_row6_col0\" class=\"data row6 col0\" >note</td>\n",
+       "      <td id=\"T_6f49e_row6_col1\" class=\"data row6 col1\" >string</td>\n",
+       "      <td id=\"T_6f49e_row6_col2\" class=\"data row6 col2\" ></td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <td id=\"T_6f49e_row7_col0\" class=\"data row7 col0\" >contact_email</td>\n",
+       "      <td id=\"T_6f49e_row7_col1\" class=\"data row7 col1\" >string</td>\n",
+       "      <td id=\"T_6f49e_row7_col2\" class=\"data row7 col2\" ></td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n"
+      ],
+      "text/plain": [
+       "<pandas.io.formats.style.Styler at 0x17750a100>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "eq_t.describe()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Updating and Deleting Data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Table rows can be modified and deleted with the SQL-like `update()` and `delete()` commands."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 32,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Inserting rows into `earthquakes`: 2 rows [00:00, 803.81 rows/s]\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "UpdateStatus(num_rows=2, num_computed_values=0, num_excs=0, updated_cols=['earthquakes.note', 'earthquakes.contact_email'], cols_with_excs=[])"
+      ]
+     },
+     "execution_count": 32,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# Add a comment to records with IDs 123 and 127\n",
```

Comment:
Maybe use a computed expression here somewhere? Right now it looks like the update dict can only do literals.

Increment some integer column by 1?